### DISCUSSION TITLE
Right usage of variants

### DISCUSSION THREAD

**Discussion by angelosne** [inventree/InvenTree#3670] [Q&A] [answered]
Hello,
I am new to Inventree and I installed it to test it on my job to help me with some builds.
I want to add [this part](https://www.ysi.com/odortu) and as you can see it has a lot of options (+conductivity, +how many meters you need).
I was about to add every one as a separate part but I stumbled on variants and as I understand is for this usage.

So I made a general ODO sensor and tagged it as "is template". After that I went to "Variants" tab and created my variants based on that one. 

Is my procedure right? Anything that can be improved? Also I'd like the general ODO sensor to not be able to be used. I marked it as inactive but I do not think this is right.

Thank you in advance

**Comment by matmair** [comment, marked as answer]
Sounds lika a reasonable aproach 👍 Deactivating the part is the approach I would take.

**Reply by matmair** [reply]
@angelosne do you need further assistance or can we mark this as answered?

**Reply by angelosne** [reply]
@matmair Yes you can close it. Currently I am not using it but in the feature I will come back and if I find a different approach I will post it!
